main.py

```
@@ -1,44 +1,113 @@
+        parse_mode='Markdown'
+    )
+
+async def button_callback(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    query = update.callback_query
+    await query.answer()
+    game_key, series_num = query.data.split('|')
+    file_id = VIDEO_DATABASE[game_key]["series"][series_num]
+    if file_id != "placeholder":
+        await query.message.reply_video(
+            video=file_id,
+            caption=f"{VIDEO_DATABASE[game_key]['name']} – Серия {series_num}"
```

Ruff (0.15.1)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)